### PR TITLE
docs: update resource recommendations in deploying docs

### PR DESCRIPTION
**Pull request by pvlugter**

### CODE REVIEW

File: akka-docs/src/main/paradox/additional/deploying.md [hunk 1]
```diff
@@ -21,7 +21,29 @@ features from Akka Management for smooth rolling updates.
  
 ### Resource limits
 
-To avoid CFS scheduler limits, it is best not to use `resources.limits.cpu` limits, but use `resources.requests.cpu` configuration instead.
+To avoid CFS scheduler limits, it is best not to use `resources.limits.cpu` limits, but use `resources.requests.cpu`
+configuration instead.
+
+For memory, it's recommended to set both `resources.requests.memory` and `resources.limits.memory` to the same value.
+The `-XX:InitialRAMPercentage` and `-XX:MaxRAMPercentage` JVM options can be used to set the heap size relative to the
+memory limit.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/additional/deploying.md:29]: Could we also mention something about off heap and that we recommend 70% MaxRAMPercentage (possible more for larger memory limit)?

**Review by patriknw — COMMENTED** [review]
looking good

File: akka-docs/src/main/paradox/additional/deploying.md (hunk 1 above)
Comment (pvlugter) [reply akka-docs/src/main/paradox/additional/deploying.md]: Ok, updated to include a 70% recommendation.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**